config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -562,7 +562,7 @@ periodics:
     workdir: true
   - org: kubernetes
     repo: kubernetes
-    base_ref: release-1.23
+    base_ref: release-1.24
```

Comment:
should we get rid of the exra refs on k/k since we established CAPZ no longer needs them? Unless cloud-provider-azure e2e test suite relies on building k8s code

Reply:
> and then we'd follow up by removing the k8s git root foo in capz's ci-entrypoint.sh

we also need to update all other tests that use ci-entrypoint.sh in test-infra to use the latest-foo settings (I'm thinking of CSI drivers mostly) as a follow up PR

Reply:
I believe this PR will do the job: https://github.com/kubernetes/test-infra/pull/26397

(to be clear, we should prioritize merging this OOT + capz PR first)